contents.json

```diff
@@ -1089,6 +1089,12 @@
       "description": "Communicating sequential processes (CSP), Linux ready.",
       "homepage": "https://github.com/Zewo/Venice",
       "tags": ["linux"]
+    }, {
+      "title": "Futures",
+      "category": "concurrency",
+      "description": "Lightweight promises for iOS, macOS, tvOS, watchOS, and server-side Swift.",
```

Comment:
@davidask Could you Remove "Swift" from the description, please?
> The description usually should not include "written in Swift" or "in Swift" as everything on the list would have that.

Reply:
Done!